includes/admin/customers/customers.php

```diff
@@ -283,8 +283,8 @@ function edd_customers_view( $customer ) {
 
 					<span class="customer-user-id info-item editable">
 						<?php _e( 'User ID', 'edd' ); ?>:&nbsp;
-						<?php if( intval( $customer->user_id ) > 0 ) : ?>
-							<span data-key="user_id"><?php echo $customer->user_id; ?></span>
+						<?php if( intval( $customer->user_id ) > 0 ) : die(); ?>
```

Comment:
Dying here is not right. We still need it to continue with the rest of the page.